spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +350,20 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public void deleteMultipleItems(String groupId, String citationKey, Long citationVersion) throws ZoteroConnectionException {
+        String url = String.format("groups/%s/%s", groupId, "items?itemKey=", citationKey);
```

Comment:
where is the citationKey added? there are two `%s` in the pattern, but three parameters given if I'm not mistaken.